domain/crossmodelrelation/state/model/remoteapplication_test.go

```diff
@@ -1279,184 +1195,77 @@ func (s *modelRemoteApplicationSuite) TestGetOffererRelationUUIDsForConsumersNon
 	c.Check(results, tc.HasLen, 0)
 }
 
-func (s *modelRemoteApplicationSuite) TestGetOffererRelationUUIDsForConsumersSameOfferMultipleConsumers(c *tc.C) {
```

Comment:
I didn't know what this was testing, but can be added back if someone cares enough.